geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/index/BaseFeatureIndex.scala

```diff
@@ -24,41 +25,57 @@ import org.opengis.filter.Filter
   * Base feature index that consists of an optional table sharing, a configurable shard, an index key,
   * and the feature id
   */
-trait BaseFeatureIndex[DS <: GeoMesaDataStore[DS, F, W], F <: WrappedFeature, W, R, C, K]
+trait BaseFeatureIndex[DS <: GeoMesaDataStore[DS, F, W], F <: WrappedFeature, W, R, C, V, K]
     extends GeoMesaFeatureIndex[DS, F, W] with IndexAdapter[DS, F, W, R, C] with LazyLogging {
 
   import org.locationtech.geomesa.utils.geotools.RichSimpleFeatureType.RichSimpleFeatureType
 
-  protected def keySpace: IndexKeySpace[K]
+  /**
+    * Primary key space used by this index
+    *
+    * @return
+    */
+  protected def keySpace: IndexKeySpace[V, K]
```

Comment:
What do the types V and K here mean?

Reply:
i think it was value and key... i'll add a scaladoc comment on the trait